Xamarin.RevenueCat.Android/Transforms/Metadata.xml

```diff
@@ -19,4 +19,5 @@
         path="/api/package[@name='com.revenuecat.purchases.common.subscriberattributes']/class[@name='ReservedSubscriberAttribute']"
         name="visibility">public
     </attr>
+    <remove-node path="/api/package[@name='com.revenuecat.purchases.paywalls']/class[@name='PaywallColor']" />
```

Comment:
As we want to use the paywalls feature, we should not remove public classes associated with that

Reply:
But that is just paywall color which we might not use it at all?

Reply:
updated with handling `PaywallColor